### ISSUE TITLE
Build error, unknown type 

### ISSUE THREAD

**Issue by dy5es41** [LedgerHQ/app-bitcoin-legacy#99]
I have my sdk defined ready as I have easily loaded other applications before, but cannot seem to load the BOLOS application for bitcoin.
Running `make load` yeilds the follow result ($COIN defaults to bitcoin):

```TARGET_NAME=TARGET_NANOS TARGET_ID=0x31100003
BOLOS_ENV=/home/dy5es41/Desktop/ledger-dev
BOLOS_ENV=/home/dy5es41/Desktop/ledger-dev
[CC]	  obj/btchip.o
In file included from src/btchip.c:20:
In file included from src/btchip_internal.h:22:
In file included from src/btchip_public_ram_variables.h:25:
include/btchip_context.h:99:5: error: unknown type name 'cx_blake2b_t'
    cx_blake2b_t blake2b;
    ^
1 error generated.
/home/dy5es41/Desktop/ledger-dev/nanos-secure-sdk//Makefile.rules_generic:51: recipe for target 'obj/btchip.o' failed
make: *** [obj/btchip.o] Error 1
```

Am I missing dependencies? How may I fix this problem? I am assuming that `cx_blake2b_t` is really defined in one of the headers

**Comment by dy5es41** [comment]
changes to wallet added in https://github.com/LedgerHQ/ledger-app-btc/commit/6f9578822acfd803b7567b74a7f33a6b5058b141#diff-c61ccd96a6d8b54d48f54a3bc4dfa7e2

commits before this compile fine

**Comment by clementperon** [comment]
Hi this app use the BLAKE_SDK available since 1.5.x.

You should update your SDK to >=1.5.x https://github.com/LedgerHQ/nanos-secure-sdk/tree/nanos-1552
